test/modules/model/SmolVLM_text_model/model.py

```diff
@@ -41,3 +42,6 @@ def get_example_inputs(self):
             "return_dict": True,
         }
         return (), kwargs
+
```

Comment:
Hmm.. Could you exclude the change of this file? Current test infra can't test this feature because the example inputs and expected circle inputs are different.

Reply:
Okay I'll remove those changes.